examples/tcpblaster/tcpblaster_client.c

```diff
@@ -37,8 +37,6 @@
  * Included Files
  ****************************************************************************/
 
-#include "config.h"
```

Comment:
This is not an error.  It's the same config.h file but for the "host".  It's copied to the examples directory during build time.
(https://github.com/apache/incubator-nuttx-apps/blob/master/examples/tcpblaster/Makefile#L112)

Reply:
Yes, you are right. Iwill remove this patch from PR.